modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -284,7 +284,13 @@
 						: document.getElementById(`cke_${editor.name}`);
 
 					if (typeof callback === 'function') {
-						callback(imageSrc);
+						callback(imageSrc, selectedItem);
+
+						instance._checkImageWidth(
+							editor,
+							editorContent,
+							imageSrc
+						);
```

Comment:
Rollback. This is needed by `adaptivemedia` version, and should not affect `itemselector` only version.